packages/semaphore-group-pcd/src/SemaphoreGroupPCD.ts

```diff
@@ -45,6 +46,9 @@ export interface SemaphoreGroupPCDArgs {
 }
 
```

Comment:
Should we also have an equivalent of SemaphoreSignaturePCD's `generateMessageHash` for SemaphoreGroupPCD? So people can send any string as a signal instead of fitting into baby jub jub field.

Reply:
Oops accidentally started a review with this comment lol

Reply:
Perhaps, I see that as out of scope for this PR.